cpp/tests/io/json/json_test.cpp

```diff
@@ -2975,4 +2977,26 @@ TEST_F(JsonReaderTest, JsonDtypeSchema)
                                  cudf::test::debug_output_level::ALL_ERRORS);
 }
 
+TEST_F(JsonReaderTest, PreprocessAndNullifyEmptyRows)
+{
+  // Test input
+  std::string const row_string = R"({"A":'TEST"'})";
+  auto string_col =
+    cudf::test::strings_column_wrapper({row_string, row_string, row_string, row_string, row_string},
+                                       {true, true, true, true, true})
```

Comment:
Where are the empty rows?

Reply:
Thanks, I've cleaned up at test and added empty rows.